README.md

```diff
@@ -8,9 +8,21 @@ This project is a demonstration of how to create a stateless Virtual JSON Graph
 clone repo
 npm install
 npm start
-open your browser and visit http://localhost:9090
+Automatically opens http://localhost:9090 on your default browser
 ```
 
+## Notes
+
+No manual refresh of browser on every change of html file.
```

Comment:
Attempt at rephrasing:

"HTML changes will refresh the browser automatically."